package-lock.json

```diff
@@ -1928,21 +1928,6 @@
           "integrity": "sha512-bN798gFfQX+viw3R7yrGWRqnrN2oRkEkUjjl4JNn4E8GxxbjtG3FbrEIIY3l8/hrwUwIeCZvi4QuOTP4MErVug==",
           "dev": true
         },
-        "color-convert": {
```

Comment:
I still have no idea what causes this shuffling around of dependencies sometimes 😵